e2e/playwright/file-tree.spec.ts

```diff
@@ -11,37 +12,115 @@ import {
 import { join } from 'path'
 import { FILE_EXT } from 'lib/constants'
 
-test.beforeEach(async ({ context, page }, testInfo) => {
+_test.beforeEach(async ({ context, page }, testInfo) => {
   await setup(context, page, testInfo)
 })
 
-test.afterEach(async ({ page }, testInfo) => {
+_test.afterEach(async ({ page }, testInfo) => {
   await tearDown(page, testInfo)
 })
 
+test.describe('integrations tests', () => {
+  test('Creating a new file or switching file while in sketchMode should exit sketchMode', async ({
+    tronApp,
+    homePage,
+    scene,
+    editor,
+    toolbar,
+  }) => {
+    await tronApp.initialise({
+      fixtures: { homePage, scene, editor, toolbar },
+      folderSetupFn: async (dir) => {
+        const bracketDir = join(dir, 'test-sample')
+        await fsp.mkdir(bracketDir, { recursive: true })
+        await fsp.copyFile(
+          executorInputPath('e2e-can-sketch-on-chamfer.kcl'),
+          join(bracketDir, 'main.kcl')
+        )
```

Comment:
`folderSetupFn` is carry over from our original `electronSetup` function. I think this another helper here is a good idea, because this could be achieved with `createProjectHelper('test-sample', 'e2e-can-sketch-on-chamfer.kcl')`